cmd/internal/find/parser/parseOptions.go

```diff
@@ -38,6 +38,37 @@ func parseOptions(args []string) (types.Options, []string, error) {
 	// Parse the args
 	err := fs.Parse(args[0:endIx])
 	if err != nil {
+		if err == flag.ErrHelp {
```

Comment:
I hope the `help` flag is the only time we'll need the optional Boolean flag support, b/c nearly all of the other flag parsing libraries I could find require posix-style flags, so `-v/--verbose` style flags. Using those libraries would mean breaking the syntactic consistency between the options and the expression operators/primaries.